README.md

```diff
@@ -11,7 +11,11 @@
 [badge-docs-main-img]: https://img.shields.io/badge/docs-main-blue
 [badge-docs-main-url]: https://famedly.github.io/rust-library-template/project_name/index.html
 
-Short description of the project.
```

Comment:
Won't block the PR on this, but changes like this should really be a separate commit.